yarn.lock

```diff
@@ -0,0 +1,157 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Please delete this file, as we only need it in the frontend folder (already exists there). Thanks!